scormxblock/static/js/src/scormxblock.js

```diff
@@ -1,209 +1,247 @@
-function ScormXBlock(runtime, element, settings) {
-    "use strict";
-    var errorCode = 0;
-
-    const commitUrl = runtime.handlerUrl(element, 'scorm_commit');
-    const ios_commitUrl = runtime.handlerUrl(element, 'scorm_ios_commit');
-    const enforce_commitUrl = runtime.handlerUrl(element, 'scorm_enforce_commit');
-    const getValueUrl = runtime.handlerUrl(element, 'scorm_get_value');
-    const syncScoreUrl = runtime.handlerUrl(element, 'sync_score_value')
-    const package_version = settings['scorm_pkg_version_value'];
-    const package_date = settings['scorm_pkg_modified_value'];
-    const ratio_value = settings['ratio_value'];
-    const open_new_tab = settings['open_new_tab_value'];
-    var timerId;
-    let pendingValues = null;
-
-    function scormInit() {
-        var $scormFrame = $('#scorm-object-frame')
-        var ratios = {
-            '4:3': 0.75,
-            '16:9': 0.5625,
-            '1:1': 1,
+//$(document).ready(
+    function ScormXBlock(runtime, element, settings) {
+        "use strict";
+        var errorCode = 0;
+    
+        const commitUrl = runtime.handlerUrl(element, 'scorm_commit');
+        const ios_commitUrl = runtime.handlerUrl(element, 'scorm_ios_commit');
+        const enforce_commitUrl = runtime.handlerUrl(element, 'scorm_enforce_commit');
+        const getValueUrl = runtime.handlerUrl(element, 'scorm_get_value');
+        const syncScoreUrl = runtime.handlerUrl(element, 'sync_score_value')
+        const package_version = settings['scorm_pkg_version_value'];
+        const package_date = settings['scorm_pkg_modified_value'];
+        const ratio_value = settings['ratio_value'];
+        const open_new_tab = settings['open_new_tab_value'];
+        var timerId;
+        let pendingValues = null;
+    
+        function scormInit() {
+            var $scormFrame = $('#scorm-object-frame')
+            var ratios = {
+                '4:3': 0.75,
+                '16:9': 0.5625,
+                '1:1': 1,
+            }
+            var resetIframeSize = function () {
+              $scormFrame.height($scormFrame.width() * ratios[ratio_value]);
```

Comment:
- The width/height of scorm container are incorrect as follow ( I tested this with "Chrome" + "Safari" ) :
![image](https://user-images.githubusercontent.com/26813045/207828124-29bfa7d7-061a-46c2-9201-f63669f09525.png)

 - If specify the height with a value :
```
        function scormInit() {
            var $scormFrame = $('#scorm-object-frame')
            var ratios = {
                '4:3': 0.75,
                '16:9': 0.5625,
                '1:1': 1,
            }
            var resetIframeSize = function () {
              $scormFrame.height(1000 * ratios[ratio_value]);
            }
            ......
```
![image](https://user-images.githubusercontent.com/26813045/207828629-1ad651a6-28d9-4647-8af2-10b175270e49.png)

 - So, I guess this issue was caused by some Edge Triggered Issue on SCORM Container width/height calculation .

Reply:
- checkout branch `hawthorn` 
```
edxapp@learning-tribes:~/venvs/edxapp/src/scormxblock-xblock$ git status
On branch hawthorn
Your branch is up-to-date with 'origin/hawthorn'.
nothing to commit, working directory clean
```
 - Ran command: `paver update_assets --settings=aws` 
 - Restarted services: `sudo /edx/bin/supervisorctl restart all`
![image](https://user-images.githubusercontent.com/26813045/207861707-7a6ee960-96b1-4b83-b2eb-b127020c206e.png)
![image](https://user-images.githubusercontent.com/26813045/207861769-0d517dd5-a999-40ef-aae1-8e287ab2c302.png)